packages/tasks/src/model-libraries.ts

```diff
@@ -172,6 +172,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/fudan-generative-vision/champ",
 		countDownloads: `path:"champ/motion_module.pth"`,
 	},
+	chatterbox: {
+		prettyLabel: "Chatterbox",
+		repoName: "Chatterbox",
+		repoUrl: "https://github.com/resemble-ai/chatterbox",
+		snippets: snippets.chatterbox,
+		countDownloads: `path:"t3_cfg.pt"`,
```

Comment:
```suggestion
		countDownloads: `path:"tokenizer.config"`,
```

I would rather count on the tokenizer config instead. Feels like a good candidate for a file that must be downloaded at every model instanciation (pytorch files can always be converted to safetensors which would break the download count). 

Other than that, looks good to me for a merge! Let's wait for a @Vaibhavs10 review though for the snippet part.